webdriver-spec.html

```diff
@@ -1259,17 +1259,11 @@
 
  <tr>
   <td>POST</td>
-  <td>/session/{<var>session id</var>}/execute/sync</td>
+  <td>/session/{<var>session id</var>}/execute</td>
   <td><a>Execute Script</a></td>
  </tr>
 
  <tr>
-  <td>POST</td>
```

Comment:
I'm actually in favor of removing `execute async`, it's totally unnecessary with promise support. I showed how it could be implemented in terms of `execute` should others want to keep it.